doc/content/source/auxkernels/Current.md

```diff
@@ -1,20 +1,40 @@
 # Current
 
-!alert construction title=Undocumented Class
-The Current has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /AuxKernels/Current
```

Comment:
This isn't in the actual PR but shows up on the website the description of current is 

Returns the electric current associated with the flux of defined species

Can we fix the grammar here to be something like 

Returns the electric current associated with the flux of a specified species

Reply:
Addressed in commit [Rephrasing 'defined species' to 'specified species'](https://github.com/shannon-lab/zapdos/pull/225/commits/323da7dd4d33cd7860a2c58308f7fe984640f90d)